test/run-test.sh

```diff
@@ -168,9 +168,10 @@ function run_code_tests {
     run_test_fail "ensure code, nanoui templates, icons unique" "find code/ nano/templates/ icons/ -type f -exec md5sum {} + | sort | uniq -D -w 32 | grep -w 'code\|nano\|icons'"
     run_test_fail "ensure code, nanoui templates, icons has no empty files" "find code/ nano/templates/ icons/ -empty -type f | grep -w 'code\|nano\|icons'"
     run_test_fail "no invalid spans" "grep -REnr --include='*.dm' \"<\s*span\s+class\s*=\s*('[^'>]+|[^'>]+')\s*>\" code/"
-    run_test_fail "code contains path as string \"/path/\"" "grep -P '\"/(obj|datum|atom|turf|area|mob)/[^\[\s\(]*\"' code/**/*.dm"
-    run_test_fail "changed files contains proc argument starting with 'var'" "grep -P '^/[\w/]\S+\(.*(var/|, ?var/.*).*\)' code/**/*.dm"
+    run_test_fail "code contains path as string \"/path/\"" "grep -RPnr '\"/(obj|datum|atom|turf|area|mob)/[^\[\s\(]*\"' code/**/*.dm"
+    run_test_fail "changed files contains proc argument starting with 'var'" "grep -RPnr '^/[\w/]\S+\(.*(var/|, ?var/.*).*\)' code/**/*.dm"
     run_test_fail "code contains alert(). Use tgui_alert() instead" "grep -RPnr '[ =\t\(,\|&]alert\(' code/**/*.dm"
+    run_test_fail "changed files contains \"src.proc()\"" "grep -RPnr --include='*.dm' '[\s\(\[!,\/=]src\.[^\.\s<\[\/,\+\-\*&\|%~>^=]*\(' code/"
```

Comment:
```suggestion
    run_test_fail "changed files contains \"src.proc()\"" "grep -RPnr --include='*.dm' '[\s\(\[!,\/=]src\.[\w\d_]*\(' code/"
```
Я немного не понимаю зачем так много здесь нужно лишних символов. Вроде как сам бьонд не даст написать функцию с большинством символов у тебя